src/checker/hunspell.rs

```diff
@@ -29,6 +29,33 @@ pub struct HunspellWrapper(pub Arc<Hunspell>);
 unsafe impl Send for HunspellWrapper {}
 unsafe impl Sync for HunspellWrapper {}
 
+// The value is `true` if string is made of emoji's
+// or unicode `VULGAR FRACTION`
+pub fn vulgar_fraction_or_emoji(word: String) -> bool {
```

Comment:
We should also avoid the copy here, maybe a `&str` would suffice as well?

Reply:
you are right! I applied the changes.